<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetBusiness</class>
 <widget class="QWidget" name="WidgetBusiness">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>577</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QDockWidget" name="dockWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>218</width>
       <height>524287</height>
      </size>
     </property>
     <property name="windowTitle">
      <string>设备列表</string>
     </property>
     <widget class="QWidget" name="dockWidgetContents">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QTreeWidget" name="treeWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="headerHidden">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>数据点</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QTableWidget" name="twYCYX">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <column>
          <property name="text">
           <string>端口编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口数据</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>是否启用</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>遥调</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QTableWidget" name="twYT">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <column>
          <property name="text">
           <string>端口编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口数据</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>是否启用</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>遥控</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTableWidget" name="twYK">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <column>
          <property name="text">
           <string>端口编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口名称</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口类型</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>端口数据</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>是否启用</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
